core/sqf/src/seatrans/tm/hbasetmlib2/src/main/java/org/trafodion/dtm/HBaseTxClient.java

```diff
@@ -452,7 +436,13 @@ public short abortTransaction(final long transactionID) throws Exception {
           Object commitDDLLock = new Object();
           synchronized(commitDDLLock)
           {
-            commitDDLLock.wait();
+             boolean loopBack = false;
+             try {
+                commitDDLLock.wait();
+             } catch(InterruptedException ie) {
+                 LOG.warn("Interrupting commitDDLLock.wait,  but retrying ", ie);
+                 loopBack = true;
+             } while (loopBack);
```

Comment:
Interesting. It is not clear why didn't Java compiler catch this issue. I am fixing the code to ignore InterruptedException.